netmanager/src/views/pages/Map/OverlayMap.js

```diff
@@ -669,7 +671,7 @@ const MapContainer = () => {
           <OverlayMap
             center={[22.5600613, 0.8341424]}
             zoom={2.4}
-            // heatMapData={heatMapData}
+            heatMapData={ENVIRONMENT === 'staging' ? heatMapData : []}
```

Comment:
@Baalmart this is where the `ENVIRONMENT` value is used. I have used strict equality(`===`) which means if the comparison between the value of the variable and the staging string is false then the heatMapData value will be an empty array always including "undefined" === 'staging' returns false

Reply:
thanks @Codebmk